packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -255,7 +252,7 @@ export class PayPalHandler extends StripeWebhookHandler {
       taxRateId = taxRate?.id;
     }
```

Comment:
Shouldn't all of this be removed as part of this effort as well?

Reply:
I believe FXA-6688 should be marked as blocked by FXA-6091.

In each of these locations the behavior of the API is actually changing:

https://github.com/mozilla/fxa/pull/14921/files#diff-e21262f459e697f72383b145f29509fa46f3f2978647507be0f88a0467d36636R245
https://github.com/mozilla/fxa/pull/14921/files#diff-e21262f459e697f72383b145f29509fa46f3f2978647507be0f88a0467d36636R255
https://github.com/mozilla/fxa/pull/14921/files#diff-e21262f459e697f72383b145f29509fa46f3f2978647507be0f88a0467d36636R331
https://github.com/mozilla/fxa/pull/14921/files#diff-bc4cc467438b7003118180f818c50d9a92f54b6f6e3e2959de0e1e8f5840f39bR614
https://github.com/mozilla/fxa/pull/14921/files#diff-bc4cc467438b7003118180f818c50d9a92f54b6f6e3e2959de0e1e8f5840f39bR621

Reply:
Thanks Julian! Confirming that I have marked this ticket as blocked and placed it back into Backlog.